src/element/binding.ts

```diff
@@ -74,8 +74,8 @@ export const bindOrUnbindLinearElement = (
     .getNonDeletedElements(onlyUnbound)
     .forEach((element) => {
       mutateElement(element, {
-        boundElementIds: element.boundElementIds?.filter(
-          (id) => id !== linearElement.id,
+        boundElements: element.boundElements?.filter(
+          (element) => element.id !== linearElement.id,
```

Comment:
we should check the `type` as well ?

Reply:
[`a1b4dd3` (#4404)](https://github.com/excalidraw/excalidraw/pull/4404/commits/a1b4dd3fb1c092e7f4e0c905d72c77b583ecf5e9#diff-3e2d80542c78b290da77a1f17fc4fbccbe43a2724e8c56c3367a7690bae592d0R82-R83)